### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by andylmfao** [domdorn/cities_skylines_2_mac_apple_silicon#2]
Lot of issues with this, just doesn't work. Is the project abandoned?

**Comment by domdorn** [comment]
yeah, havent got it to work after newer patches came out.. at one point, I gave up trying. will mark the project as archived

**Comment by sguessone** [comment]
what a shame though, I had just found this repo 🥺

**Comment by domdorn** [comment]
### Project is dead as stated on top of the README.

@ExtRIELICi sorry, no updates.. I'm using a cloud service "g***** now" for now -.- its sad, I cant utilize this hardware for this anymore.
